INSTALLATION.md

```diff
@@ -183,6 +205,34 @@ This guide is for `VSCode` users to easily manage their `mongoDB` databases:-
 
 <br/>
 
+### Instructions to edit records for ADMIN user
+
+**Note**: You can skip these instructions for now if you don't have running instance of Talawa-Admin.
+
+#### I. MongoDB Compass
+
+1. Open a terminal and run `mongod` command.
+2. Open MongoDB Compass and click on `Connect`.
+3. Select `user` collections and edit the data. Change:
+     1. `userType` from ADMIN to SUPERADMIN
+     2. `adminApproved` from false to true
+     
+     * ![Screenshot_251](https://user-images.githubusercontent.com/121368112/218772135-da412ac2-042f-4695-9a84-df3f39b4b0d2.png)
+
+#### II. Mongo Shell
+
+1. Open a terminal and run `mongod` command.
+2. Open a separate terminal and run `mongosh` command.
+3. You can use the following command to edit the `user` collections and edit the data:
+
+   ```
+    db.users.updateOne({userType: "ADMIN"}, {$set: {userType: "SUPERADMIN", adminApproved: true}})
```

Comment:
1. This is too broad. It makes all `ADMIN` users `SUPERADMIN`
2. How can you find the existing `ADMIN` and how can you specifically update that user?